streaming/base/partition/pynum.py

```diff
@@ -87,10 +87,10 @@ def _get_partitions_pynum_padded(dataset_size: int,
     # Of those sample IDs that remain (which need to be present to keep samples balanced across
     # ranks), reassign the ones that extend past the end of the dataset to the last sample (as we
     # were probably on the last shard, in order to avoid an extra shard download).
-    start = unpadded_samples_per_rank
-    stop = samples_per_rank
-    ids[:, :, start:stop] = np.where(ids[:, :, start:stop] < dataset_size, ids[:, :, start:stop],
-                                     dataset_size - 1)
+    i = unpadded_samples_per_rank
```

Comment:
`start` and `stop` variable name was more meaningful than `i` and `j`. Can we please revert the name again ?